deploy-android-app/action.yml

```diff
@@ -0,0 +1,193 @@
+name: Deployment
+author: Petar Kramaric
+description: GitHub action for deploying android applications
+inputs:
+  github_token:
+    description: A GitHub personal access token with repo scope to access private repositories.
+    required: true
+  branch:
+    description: Which branch is running the job
+    required: true
+    type: choice
+    options:
+      - main
```

Comment:
is this used?

Reply:
yes i am suppose to validate that it is the main branch. So I will add that to validation